ui/css/perf.css

```diff
@@ -106,10 +106,13 @@ to { opacity: 0; }
     background-image: url("../img/tip-locked.png");
 }
 
-#close-popup {
-  float:right;
-  display:inline-block;
-  cursor:pointer;
+.graphchooser-close {
+  color: #fff;
+  outline: none;
+}
+
+.graphchooser-close:hover {
+    color: #fff;
```

Comment:
Could you put these changes into a seperate commit (with its own descriptive title like "Use bootstrap-style close button for graph popup)? Also, again, most of these CSS properties shouldn't be necessary since bootstrap provides them. Just add the color tags (and the outline if necessary?).